tests/api/test_api_ui_collection_viewsets.py

```diff
@@ -14,29 +14,3 @@ def setUp(self):
         patcher = mock.patch("galaxy_pulp.GalaxyCollectionVersionsApi", spec=True)
         self.versions_api = patcher.start().return_value
         self.addCleanup(patcher.stop)
-
```

Comment:
Please write unit tests for your changes.

Reply:
@cutwater Unit tests will be added in https://github.com/ansible/galaxy-api/pull/121 so this PR is not blocked.

I'll rebase #121 after this PR is merged.